web/annotation_layer_builder.css

```diff
@@ -93,6 +93,64 @@
   width: 115%;
 }
 
+.annotationLayer .checkboxWidgetAnnotation label,
+.annotationLayer .radioButtonWidgetAnnotation label {
+  background-color: rgb(222, 229, 255);
```

Comment:
When the pdf is pre-filled, the checked box is drawn in the canvas. I changed the color to be opaque so that the checked box is hidden behind the annotation.